src/Spectre.Console/Enrichment/ProfileEnricher.cs

```diff
@@ -2,22 +2,22 @@ namespace Spectre.Console.Enrichment;
 
 internal static class ProfileEnricher
 {
-    private static readonly List<IProfileEnricher> _defaultEnrichers = new List<IProfileEnricher>
-        {
-            new AppVeyorEnricher(),
-            new BambooEnricher(),
-            new BitbucketEnricher(),
-            new BitriseEnricher(),
-            new ContinuaEnricher(),
-            new GitHubEnricher(),
-            new GitLabEnricher(),
-            new GoCDEnricher(),
-            new JenkinsEnricher(),
-            new MyGetEnricher(),
-            new TeamCityEnricher(),
-            new TfsEnricher(),
-            new TravisEnricher(),
-        };
+    private static readonly List<IProfileEnricher> _defaultEnrichers = new()
+    {
+        new AppVeyorEnricher(),
+        new BambooEnricher(),
+        new BitbucketEnricher(),
+        new BitriseEnricher(),
+        new ContinuaEnricher(),
+        new GitHubEnricher(),
+        new GitLabEnricher(),
+        new GoCDEnricher(),
+        new JenkinsEnricher(),
+        new MyGetEnricher(),
+        new TeamCityEnricher(),
+        new TfsEnricher(),
+        new TravisEnricher(),
+    };
```

Comment:
Couldn't this be just a plain array since the list is never updated?

```suggestion
    private static readonly IProfileEnricher[] _defaultEnrichers =
    {
        new AppVeyorEnricher(),
        new BambooEnricher(),
        new BitbucketEnricher(),
        new BitriseEnricher(),
        new ContinuaEnricher(),
        new GitHubEnricher(),
        new GitLabEnricher(),
        new GoCDEnricher(),
        new JenkinsEnricher(),
        new MyGetEnricher(),
        new TeamCityEnricher(),
        new TfsEnricher(),
        new TravisEnricher(),
    };
```